hello-world-apb/roles/deprovision-hello-world-apb/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+- openshift_v1_route:
+    name: hello-world
+    namespace: '{{ namespace }}'
+    state: absent
+
+- k8s_v1_service:
+    name: hello-world
+    namespace: '{{ namespace }}'
+    state: absent
+
+- openshift_v1_deployment_config:
```

Comment:
You'll probably need to scale down and delete the replication controller. I updated the deprovision generation script to do that automatically if you want to take a look: 
https://hastebin.com/ohubutuqad.py